packages/affiliates/dappmining-config.example.js

```diff
@@ -0,0 +1,26 @@
+const moment = require("moment");
+
+// Emp to process rewards for
+const empAddress = "0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5";
+
+// Start of payout period
+const startTime = moment("2020-12-10", "YYYY-MM-DD").valueOf();
+// End of payout period
+const endTime = moment("2020-12-28", "YYYY-MM-DD").valueOf();
+// Total amount of rewards to payout for period
+const totalRewards = 25000;
+
+// Default reward address for emp create events which are untagged
+const defaultAddress = "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35";
```

Comment:
Are these reward addresses sponsors or deployers? Might be worth clarifying that

Reply:
Yes will update. This is the dapp developer address to receive rewards by default when no tag is specified on transaction